internal/distro/fedora/partition_tables.go

```diff
@@ -155,27 +155,17 @@ var iotBasePartitionTables = distro.BasePartitionTableMap{
 		},
 	},
 	distro.Aarch64ArchName: disk.PartitionTable{
-		UUID: "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
-		Type: "gpt",
+		UUID: "0xc1748067",
+		Type: "dos",
```

Comment:
just an heads up, this makes any other arm device that I have ubootable - i.e. on a rockpro64 this doesn't boot, but it boots using the "old" method

Reply:
I'll tset with https://github.com/osbuild/osbuild-composer/pull/3374 perhaps?

Reply:
@runcom Yes please, I'm doing the same.